src/main/java/juja/sqlcmd/Application.java

```diff
@@ -22,21 +23,139 @@ public static void main(String[] args) {
+            System.err.println("Username is null.");
+        }
+        try (Statement statement = connection.createStatement()) {
+            String query = String.format("DELETE FROM \"user\" WHERE name = '%s'", userName);
+            statement.executeUpdate(query);
+        }
+    }
 
+    /**
+     * Method changes the username in the "name" field in the "user" table.
+     *
+     * @param oldName old user name. It will be overwritten with the new value.
+     * @param newName new user name value.
+     * @throws SQLException if a database access error occurs
+     */
+    private void changeUserName(String oldName, String newName) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            String query = String.format("UPDATE \"user\" SET name = '%s' WHERE name = '%s'", newName, oldName);
+            statement.executeUpdate(query);
+        }
+    }
 
+    /**
+     * The method prints the table data to the console in the format Field1 | Field2 | ... | FieldN
+     *
+     * @param tableName the name of the table whose data should be output to the console.
+     * @throws SQLException if a database access error occurs
+     */
+    private void showTableData(String tableName) throws SQLException {
+        if (tableName == null) {
+            System.err.println("Can't display data. Table Name is Null");
+            return;
+        }
+        System.out.println(getTableData(tableName));
+    }
 
+    /**
+     *The method returns all data from the table.
+     *
+     * @param tableName the name of the table whose data should be returned
+     * @return all data from the table in the format Field1 | Field2 | ... | FieldN
+     * @throws SQLException if a database access error occurs
+     */
+    private String getTableData(String tableName) throws SQLException {
+        String query = "SELECT * FROM \"" + tableName + "\"";
+        try (Statement statement = connection.createStatement();
+             ResultSet resultSet = statement.executeQuery(query)) {
+            int columnCount = resultSet.getMetaData().getColumnCount();
+            StringBuilder result = new StringBuilder();
+            while (resultSet.next()) {
+                for (int i = 1; i <= columnCount; i++) {
+                    result.append(resultSet.getString(i))
+                            .append(" | ");
+                }
+                result.replace(result.length() - 3, result.length(), "\n");
+            }
+            return result.toString();
+        }
     }
 
+    /**
+     * Method adds users to the table "user".
+     *
+     * @param nameAndPassword array of users and their passwords.
+     *                        One element of array it is one user and his password.
+     *                        One element of array has format username=password.
+     * @throws SQLException if a database access error occurs
+     */
+    private void addUsers(String[] nameAndPassword) throws SQLException {
```

Comment:
Try to rename variable in paramaters. I think nameAndPassword a little not what you need.

Reply:
Maybe you're right. I'll be renaming this variable